test/suite/test_cursor_bound01.py

```diff
@@ -108,7 +108,42 @@ def test_bound_api(self):
         cursor.set_key(self.gen_key(1))
         cursor.bound("action=set,bound=lower")
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda: cursor.largest_key(),
-            '/Invalid argument/')
+            '/setting bounds is not compatible with cursor largest key/')
+
+        # Check edge cases with bounds config
+        cursor.set_key(self.gen_key(1))
+        # Setting the bound without providing an action works.
+        self.assertEqual(cursor.bound("bound=lower"), 0)
+        cursor.reset()
+
+        cursor.set_key(self.gen_key(1))
+        # Setting an action without a bound won't work.
+        self.assertRaisesWithMessage(wiredtiger.WiredTigerError, lambda: cursor.bound("action=set"),
+            '/a bound must be specified when setting bounds, either "lower" or "upper"/')
+        cursor.reset()
+
+        cursor.set_key(self.gen_key(1))
+        # Giving a longer config string works, WT_PREFIX_MATCH will accept it.
+        self.assertEqual(cursor.bound("action=setting, bound=lower"), 0)
+        cursor.reset()
+
+        cursor.set_key(self.gen_key(1))
+        cursor.bound("action=set,bound=lower")
+        # Giving a longer config string works, WT_PREFIX_MATCH will accept it.
+        self.assertEqual(cursor.bound("action=clearing"), 0)
+        cursor.reset()
+
+        cursor.set_key(self.gen_key(1))
+        cursor.bound("action=set,bound=lower")
```

Comment:
I think this line got duped and can be removed let me know if I'm misunderstanding otherwise

Reply:
The idea was to have a lower bound set for an "action" thats not setting the bound to then be called, but that's not been made a hard rule throughout this test, and the line can be removed.